### ISSUE TITLE
VS Code Server Fails to start on Alpine Linux

### ISSUE THREAD

**Issue by feamsr00** [microsoft/vscode-remote-release#1429]
- VSCode Version: 1.38.1
- Local OS Version: 10.0.18362.356 (b1903)
- Remote OS Version: Alpine Linux 3.10.2
- Remote Extension/Connection Type: WSL

Steps to Reproduce:

1. Start VS Code 
2. Cmd: Remote WSL: New Window
3. After installation steps it fails to launch the service

Does this issue occur when you try this locally?: N/A?
Does this issue occur when you try this locally and all extensions are disabled?: N/A? (disabled all exts and it still fails to launch)

Terminal Log
```
[2019-09-16 17:43:05.186] Starting VS Code Server inside WSL (Alpine-Base)
[2019-09-16 17:43:05.187] Extension version: 0.39.5, Windows build: 18362. Multi distro support: enabled. WSL path support: enabled
[2019-09-16 17:43:05.606] Launching C:\WINDOWS\System32\wsl.exe -d Alpine-Base sh -c '"$VSCODE_WSL_EXT_LOCATION/scripts/wslServer.sh" b37e54c98e1a74ba89e03073e5a3761284e3ffb0 stable .vscode-server 0  ' in c:\exec\scoop\apps\vscode-portable\1.38.1\data\extensions\ms-vscode-remote.remote-wsl-0.39.5
[2019-09-16 17:43:06.208] WSL version: 4.4.0-18362-Microsoft Alpine-Base
[2019-09-16 17:43:06.208] Updating server...
[2019-09-16 17:43:06.508] Starting server: /root/.vscode-server/bin/b37e54c98e1a74ba89e03073e5a3761284e3ffb0/server.sh  --port=0 --fileWatcherPolling=0 
[2019-09-16 17:43:06.509] 
[2019-09-16 17:43:06.509] * Note: Support for Alpine Linux is in preview.
[2019-09-16 17:43:06.545] /root/.vscode-server/bin/b37e54c98e1a74ba89e03073e5a3761284e3ffb0/server.sh: line 12: /root/.vscode-server/bin/b37e54c98e1a74ba89e03073e5a3761284e3ffb0/node: not found
[2019-09-16 17:43:06.545] VS Code Server for WSL closed unexpectedly.
[2019-09-16 17:43:06.546] For help with startup problems, go to
[2019-09-16 17:43:06.546] https://code.visualstudio.com/docs/remote/troubleshooting#_wsl-tips
```

I do recognize that Alpine is in preview. Do you need any additional Feedback? How do I troubleshoot this further?

Thanks!

**Comment by WSLUser** [comment]
Did you try the Nightly version? Also, did you install Insiders for VSCode? Test those and then check for the log output. Look for the extension under Output. Also, the end of your snippet tells you how to troubleshoot further.

**Comment by aeschli** [comment]
Where did you get `Alpine-Base` from?
We currently only support Alpine from the Windows Store.

**Comment by feamsr00** [comment]
I got Alpine from https://github.com/yuk7/AlpineWSL which in turn just wraps http://dl-cdn.alpinelinux.org/alpine/v3.10/releases/x86_64/alpine-minirootfs-3.10.2-x86_64.tar.gz. So its stock. What are the changes that make the WinStore version of Alpine special in the userspace? Could I make those changes myself?

**Comment by aeschli** [comment]
The Remote-WSL extension hardcodes the name. I will fix it.

**Comment by aeschli** [comment]
Fixed in 0.39.7 (requires latest insider build)
